javascript/javascript1/week4/script2.js

```diff
@@ -0,0 +1,46 @@
+//Vowel Count
+
+function vowelCount(letters) {
+  let count = 0;
+  for (let i = 0; i < letters.length; i++) {
+    let letter = letters.charAt(i).toLocaleLowerCase();
+    if (
+      letter === "a" ||
+      letter === "e" ||
+      letter === "i" ||
+      letter === "o" ||
+      letter === "u"
+    ) {
+      count++;
+    } else {
+      count = count;
+    }
+  }
+  return console.log(count);
+}
+let inputString =
+  "The charAt method returns the character at the specified index in a string. The index of the first character is 0, the second character is 1, and so on";
+vowelCount(inputString);
+vowelCount(prompt("please write your paragraph"));
+
+//Digit*Digit
+
+function multiply(arrgm) {
```

Comment:
Try to give a bit more meaningful names. Since it is numbers you can give something that shows its an number